tgui/packages/tgui/interfaces/Pandemic/Symptom.tsx

```diff
@@ -15,12 +15,12 @@ export const SymptomDisplay = (props, context) => {
   return (
     <Section fill title="Symptoms">
       {symptoms.map((symptom) => {
-        const { name, desc, threshold_desc } = symptom;
+        const { name, desc, focus_desc, threshold_desc } = symptom;
         return (
           <Collapsible key={name} title={name}>
             <Stack fill>
               <Stack.Item grow={3}>
-                {desc}
+                {desc + focus_desc}
```

Comment:
This should probably be its own item in the stack.

Reply:
Making it a collapsible seems unnecessary considering it's a single line of text. I've also had problems trying to separate the items and more problems trying to get them to be on different lines.

You likely know more javascript and typescript than me, but seriously, I can't do much more. I've already tried multiple times.